src/foam/nanos/auth/PermissionedPropertyDAO.js

```diff
@@ -260,8 +227,12 @@ foam.CLASS({
     {
       name: 'put',
       javaCode: `
-        FObject fo = ((FObject) obj).fclone();
-        getDelegate().put(dao.maybeRemoveProperties(getX(), fo), sub);
+  FObject oldObj = dao.getDelegate().find(((FObject) obj).getProperty("id"));
```

Comment:
This regresses back to a find on every put to the sink. @kgrgreer fixed this yesterday.

Reply:
Yes, keep the current code. for this method.

Reply:
sorry my bad, copy paste error.